pkg/constants/constants.go

```diff
@@ -61,6 +61,11 @@ const (
 	// DefaultEnvoyImage is the default envoy proxy sidecar image if not defined in the osm MeshConfig
 	DefaultEnvoyImage = "envoyproxy/envoy-alpine:v1.18.3"
 
+	// DefaultEnvoyWindowsImage is the default envoy proxy windows sidecar image if not defined in the osm MeshConfig
+	// TODO(davinci26): This should be updated to the nanoserver based image when it becomes available
```

Comment:
Can you include a reference to an OSM issue corresponding to this work?

Reply:
Will add an issue. Do you do code TODO without issues in general?

Reply:
// TODO(#<issue>): ...

Reply:
fixed now, Delyan had some TODO with different formats and they appeared first on my search so I got the format wrong